content/en/docs/develop/core-modules/evm.md

```diff
@@ -0,0 +1,935 @@
+---
+title: EVM
+weight: 170
```

Comment:
the documents are alphabetically ordered. `weight` should be `90` and other module pages are also needed to be updated accordingly.

Reply:
https://github.com/xpladev/docs/pull/14/commits/35fe8b84d38b932eda47b07fa551eeaba4a10195